Makefile

```diff
@@ -17,26 +17,26 @@ docker-build: check-version
 		-t mesg/engine:$(MAJOR_VERSION) \
 		-t mesg/engine:$(MINOR_VERSION) \
 		-t mesg/engine:$(PATCH_VERSION) \
+		-t mesg/engine:dev \
```

Comment:
Are we sure we want to tag it dev? this version is only for build on the dev branch, not local and not prod.